src/App.css

```diff
@@ -1,39 +1,43 @@
-.App {
-  text-align: center;
+.Calculator {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  grid-template-rows: repeat(6, 1fr);
+  width: 500px;
+  gap: 1px;
+  padding: 1rem;
```

Comment:
- [ ] Please adjust the grid system you built for the calculator layout to be more consistent 😄 Yours looks great, but a few adjustments to follow the given design would make it even better 😉 💪 Kindly adjust the 7, 4, and 1 buttons!
**Yours**
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/134216422-352fd475-a7e7-4b44-bc3d-0e4e3b0dfac5.png)
**Original**
<img width="558" alt="calculator" src="https://user-images.githubusercontent.com/47625407/134216489-55630f44-3614-4436-9282-e410031762e9.png">

Reply:
Resolved